joblib/pool.py

```diff
@@ -363,14 +351,12 @@ def get():
 
         self.get = get
 
-        if self._reducers:
```

Comment:
Why removing this case ? I don't fully understand this peace of code but it seems to dump the full object in a memory buffer. Hence it could use a lot of memory.